### ISSUE TITLE
undefined method `env' for #<ActionCable::Channel::ConnectionStub>

### ISSUE THREAD

**Issue by iDuuck** [appsignal/appsignal-ruby#684]
## Describe the bug

I am currently trying to make my RSpec test working, and I recently stumbled upon a failure, which I think is a bug in the system.

When I use the `stub_connection` helper. I am getting the following error.

```
#<NoMethodError: undefined method `env' for #<ActionCable::Channel::ConnectionStub:0x00007f81883a57d0>
Did you mean?  end>
```

## To Reproduce

- Using AppSignal for Ruby gem version 2.11.2
- In my app using framework/library/gem ... version 6.0.3.2
- With this code:
  ```
  context 'with a valid pubsub token' do
      let(:user) { create(:user) }

      before do
        stub_connection(identified_as: :user, current_user: user, token: <TOKEN-HERE>)
      end

      it 'confirms the subscription' do
        subscribe

        expect(subscription.confirmed?).to eq(true)
      end
    end
  ```

**Comment by tombruijn** [comment]
I've reproduced the problem and am working on a fix.

**Comment by tombruijn** [comment]
@iDuuck Thanks again for the report! A fix for this has been released in Ruby gem 2.11.7. Please upgrade and let us know if you run into any problems :)
